style.css

```diff
@@ -35,6 +35,43 @@ nav {
   width: 24px;
 }
 
+.mobileMenu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: none;
+  background-color: #2b2a2a;
+  width: 100%;
+  height: 1000px;
+  padding: 24px;
+}
```

Comment:
- [ ] Please try to make the design of the pop-up window similar to that in the Figma template. The font size of the text should be larger. The text underlines should be thinner and the close button ought to be smaller 👍 
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/174246535-040d151a-00ad-439f-b322-bb12dec26389.png)
### Required 
![image](https://user-images.githubusercontent.com/45460192/174246696-e8ff00bc-fa4c-4607-b618-b08a74642bfa.png)

Reply:
- [ OPTIONAL] you've made changes from the previous reviewer, but again I recommend to also change your white color to the one from Figma design.

Reply:
@[ishimwezachee](https://github.com/ishimwezachee)
color changed